test.py

```diff
@@ -118,3 +120,39 @@ def test_nested_input():
 def test_function_mixup():
     """Make sure that different functions with the same args are cached under different keys"""
     assert get_random_with_args(1) != get_another_random_with_args(1)
+
+
+def callTextChangeTest():
+    process = Popen(["python", "textChangeTest.py"], stdout=PIPE)
```

Comment:
PEP8 - filenames (modules) should be snake_case. https://www.python.org/dev/peps/pep-0008/#package-and-module-names